.github/workflows/dissect-ci-template.yml

```diff
@@ -71,34 +71,32 @@ jobs:
 
   test:
     needs: build
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     continue-on-error: ${{ matrix.allow_failure }}
     strategy:
       fail-fast: false
       matrix:
+        os: ["ubuntu-latest", "windows-latest"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
         include:
-          - python-version: "3.9"
-            python-include: "python3.9"
-            tox-env: "py39"
-            allow_failure: false
-          - python-version: "3.10"
-            python-include: "python3.10"
-            tox-env: "py310"
-            allow_failure: false
-          - python-version: "3.11"
-            python-include: "python3.11"
-            tox-env: "py311"
-            allow_failure: false
+          - allow_failure: false
+            tox-version: "4.2.4"
+            include-prefix: "python"
           - python-version: "3.12"
-            python-include: "python3.12"
-            tox-env: "py312"
             allow_failure: true
-          - python-version: "pypy3.9"
-            python-include: "pypy3.9"
-            tox-env: "pypy39"
+          - os: "windows-latest"
+            # There is a bug in tox < 4.4.0 (and Python 3.9) on Windows where parsing
+            # commandlines from tox.ini files fails. This results in a failure when
+            # using tox' envtmpdir in a pytest commandline.
+            tox-version: "4.4.0"
+            allow_failure: true
+          - os: "ubuntu-latest"
+            python-version: "pypy3.9"
             allow_failure: false
```

Comment:
It didn't seem to do so. I can test once more because it seems odd.

Reply:
Nope, it doesn't work. Apparently because you override 1 of the matrix values (python-version), it will create a whole new entry which is not modified by any of the other include items.